### ISSUE TITLE
Cannot run app after upgrading to react native 0.64

### ISSUE THREAD

**Issue by malcolmmcswain** [square/in-app-payments-react-native-plugin#117]
### Describe the issue
After running npm upgrade to react-native ^0.63.4 and react-native-square-in-app-payments ^1.5.0 I get an error that causes the app to crash:

```
BUNDLE  ./index.js

error: node_modules\react-native-square-in-app-payments\src\ApplePay.js: function (path) {
          return fn.call(state, path, state);
        } could not be cloned.
```

### Expected behavior
App runs and behaves normally

**Environment (please complete the following information):**
  - platform: Android
  - OS and version: Android 9 (API 28)
  - dev environment: Windows
  - In-App Payments Plugin version: 1.5.0

react-native info output:
```
System:
    OS: Windows 10 10.0.19041
    CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
    Memory: 12.42 GB / 31.95 GB
  Binaries:
    Node: 12.18.3 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK: Not Found
  IDEs:
    Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6626763
    Visual Studio: 16.7.30330.147 (Visual Studio Community 2019)
  Languages:
    Java: 11.0.8 - C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
    Python: 3.8.5 - C:\Python38\python.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^17.0.2 => 17.0.2
    react-native: ^0.63.4 => 0.63.4
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

**Comment by hukid** [comment]
@malcolmmcswain can you run the quick start project successfully? the project has been upgraded to `0.64` and it works fine.

**Comment by vishwajitshep0815** [comment]
The latest PR merged is using React Native "0.64.1". So, closing the issue.
